index.html

```diff
@@ -43,33 +48,61 @@
 </div>
 
 
-<section class="Hallelujah">
+<section class="hallelujah">
     
-    <section class="Works">
-        <div class="Project-name"><h2>Projects</h2></div>
+    <section class="works">
+        <div class="project-name"><h2>Projects</h2></div>
         <div class="work one">
-            <div class="project photo"><img src="assets/work/work_one.png" class="work_one" alt="#"></div>
-            <div class="project summary">
-                <div class="nameP"><h3>Project name goes here</h3></div>
```

Comment:
Please ensure you replicate the design as much as possible (e.g., font, colors, images, buttons, margins) [using the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9). Please ensure to align` projects` as in the Figma template.

Your Implementation | Figma Design
:---:|:---:
![your-design](https://user-images.githubusercontent.com/10332499/149752679-1ef931ab-e61d-4321-95a2-e7e2d0b55921.PNG) | ![figma](https://user-images.githubusercontent.com/10332499/149752733-b6c29ab1-ec6f-4145-8155-4dd34d9a5fec.PNG).